packages/common/src/nominator/nominator.ts

```diff
@@ -7,10 +7,18 @@ import { ChainData, Constants, queries, Types } from "../index";
 import logger from "../logger";
 import EventEmitter from "eventemitter3";
 import { sendProxyDelayTx, sendProxyTx } from "./NominatorTx";
+import { getNominatorChainInfo } from "./NominatorChainInfo";
 
 export const nominatorLabel = { label: "Nominator" };
 
 export interface NominatorStatus {
+  state?:
+    | "Nominated"
```

Comment:
please add a enum or a type for this, we should not rely on strings like that all over

Reply:
Ideally you should be able to use it at NominatorState.Nominated, NominatorState.Stale, etc.
or something similar